app/src/main/java/com/sdp/movemeet/backend/serialization/ActivitySerializer.java

```diff
@@ -23,7 +23,7 @@
     // The key used to access the numberParticipant attribute of a serialized Activity
     public static final String NPART_KEY = "numberParticipant";
     // The key used to access the participantsId attribute of a serialized Activity
-    public static final String PARTICIPANTS_KEY = "participantsId";
```

Comment:
Yes the annoying bugs !

Reply:
Yes, that is it! Because of this little "s" that didn't match the "participantId" field on Firebase Firestore, we couldn't in fact access the IDs of the participants! This is now fixed!